Microsoft.Azure.Cosmos.Samples/Tools/CTL/Scenarios/ChangeFeedPullScenario.cs

```diff
@@ -0,0 +1,183 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace CosmosCTL
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Net;
+    using System.Diagnostics;
+
+    using App.Metrics;
+    using App.Metrics.Counter;
+    using App.Metrics.Gauge;
+
+    using Microsoft.Azure.Cosmos;
+    using Microsoft.Extensions.Logging;
+
+    using Newtonsoft.Json;
+    
+
+    internal class ChangeFeedPullScenario : ICTLScenario
+    {
+        private InitializationResult initializationResult;
+
+        public async Task InitializeAsync(
+            CTLConfig config,
+            CosmosClient cosmosClient,
+            ILogger logger)
+        {
+            this.initializationResult = await CreateDatabaseAndContainerAsync(config, cosmosClient);
+            if (this.initializationResult.CreatedDatabase)
+            {
+                logger.LogInformation("Created database for execution");
+            }
+
+            if (this.initializationResult.CreatedContainer)
+            {
+                logger.LogInformation("Created collection for execution");
+            }
+
+            if (config.PreCreatedDocuments > 0)
+            {
+                logger.LogInformation("Pre-populating {0} documents", config.PreCreatedDocuments);
+                await Utils.PopulateDocumentsAsync(config, logger, new List<Container>() { cosmosClient.GetContainer(config.Database, config.Collection) });
+            }
+        }
+
+        public async Task RunAsync(
+            CTLConfig config,
+            CosmosClient cosmosClient,
+            ILogger logger,
+            IMetrics metrics,
+            string loggingContextIdentifier,
+            CancellationToken cancellationToken)
+        {
+            Stopwatch stopWatch = new Stopwatch();
```

Comment:
can be simplified:
```suggestion
            Stopwatch stopWatch = Stopwatch.StartNew();
```

And remove the Start line below